### DISCUSSION TITLE
How to stop getting updates for STS when I want to stay on LTS

### DISCUSSION THREAD

**Discussion by paulomorgado** [NuGet/Home#13959] [Q&A] [answered]
Now that .NET 9.0 has been released, how do I stop getting updates to 9.0 libraries when I need to stay on 8.0?

I know 9.0 libraries have 8.0 targets, but with different support lifecycles.

**Comment by paulomorgado** [comment]
Using:

```xml
<PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="[8.0.2,9.0.0)" />
```

instead of:

```xml
<PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.2" />
```

Does not report an available update to 9.0.0.

But, if I use:

```xml
<PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="[8.0.0,9.0.0)" />
```

It will offer an update to 8.0.2, but will change the declaration to:

```xml
<PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.2" />
```

and then offer an update to 9.0.0

**Comment by jeffkl** [comment, marked as answer]
Closing in favor of https://github.com/NuGet/Home/issues/5725 which is the feature request that would provide this functionality.
